src/IntelliTect.Coalesce/TypeDefinition/MethodViewModel.cs

```diff
@@ -176,7 +176,7 @@ public string CsParameters
         public string DisplayName =>
             this.GetAttributeValue<DisplayNameAttribute>(a => a.DisplayName) ??
             this.GetAttributeValue<DisplayAttribute>(a => a.Name) ??
-            Name.ToProperCase();
+            Name.ToProperCase()!;
```

Comment:
Is null forgiveness needed here? Name is declared non-nullable, and ToProperCase() does not return null if the parameter is not null.